### PR TITLE
[4.0] Remove unused imports

### PR DESCRIPTION
**Pull request by SharkyKZ**
### Summary of Changes

Followup to #26332. This applies PHP-CS-Fixer's `no_unused_imports` rule across the CMS.

### Testing Instructions

Code review.

### Documentation Changes Required

No.

### CODE REVIEW

File: tests/Integration/DBTestHelper.php [hunk 1]
```diff
@@ -8,10 +8,7 @@
  */
 namespace Joomla\Tests\Integration;
 
-use Joomla\Database\DatabaseDriver;
 use Joomla\Database\DatabaseFactory;
-use Joomla\Database\Mysqli\MysqliDriver;
-use Joomla\Tests\Integration\IntegrationTestCase;
```
Comment (Quy) [inline tests/Integration/DBTestHelper.php:14]: Revert? See line 44.

Comment (SharkyKZ) [reply tests/Integration/DBTestHelper.php:14]: This file and `IntegrationTestCase` are in the same namespace.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 7c71455fff1114e68027e1f3933411545a5afeda<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/26337">issues.joomla.org/tracker/joomla-cms/26337</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Pull request merged**